Documentation/source/guide/tutorials-evaluating-script-structure.rst

```diff
@@ -0,0 +1,208 @@
+.. for help on writing/extending this file, see the reStructuredText cheatsheet
+   http://github.com/ralsina/rst-cheatsheet/raw/master/rst-cheatsheet.pdf
+   
+   .. _evaluating-script-structure:
+
+
+Thanks to Martin Constable to allow us to use his documents.
```

Comment:
Would add Martin (and you) to the Documentation authors (https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTORS.txt and in the intro)